assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,56 @@
 console.log('***** Music Collection *****')
+
+// Created a variable empty array called collection
+let collection = [];
+
+// function addToCollection, with parameters for title, artist, yearPublished
+// function takes in album objects and pushes into collection array
+// added an album to the collection with objects
```

Comment:
Good call bringing the requirements over from the readme so they are easier to keep track off.